lib/plug/upload.ex

```diff
@@ -161,7 +161,7 @@ defmodule Plug.Upload do
   defp i(integer), do: Integer.to_string(integer)
 
   defp path(prefix, tmp) do
-    {_mega, sec, mili} = :erlang.now
+    {_mega, sec, mili} = :os.timestamp
```

Comment:
We actually want a unique value here so `:os.timestamp` is wrong. We should probably keep using `:erlang.now` until 18 is out and then use `:erlang.unique_integer` with this http://www.erlang.org/documentation/doc-7.0-rc1/erts-7.0/doc/html/time_compat.erl.